external-crates/move/move-compiler/src/typing/translate.rs

```diff
@@ -2306,6 +2317,23 @@ fn gen_unused_warnings(context: &mut Context, mdef: &T::ModuleDefinition) {
         .env
         .add_warning_filter_scope(mdef.warning_filter.clone());
 
+    for (loc, name, c) in &mdef.constants {
+        context
+            .env
+            .add_warning_filter_scope(c.warning_filter.clone());
+
+        if !context.used_consts.contains(name) {
+            let msg = format!("The constant '{name}' is never used. Consider removing it.");
```

Comment:
Nit: these errors don't use punctuation in most cases.

```suggestion
            let msg = format!("The constant '{name}' is never used");
```

"Consider removing it" is implied -- it might also mean someone called the wrong name, etc., so I'm tempted to not suggest specific remediation.

Reply:
I chose this phrasing to be consistent with other warnings (e.g., the one for unused aliases) that pre-date this PR.